projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/review-submit/review-submit.component.spec.ts

```diff
@@ -85,8 +85,14 @@ describe('ReviewSubmitComponent', () => {
       loadSupportedDeliveryModes: createSpy()
     };
     mockCartService = {
-      activeCart$: new BehaviorSubject(null),
-      entries$: new BehaviorSubject(null)
+      cart$: new BehaviorSubject(null),
+      entries$: new BehaviorSubject(null),
```

Comment:
This and `cart$` don't represent the actual class correctly - we don't have public properties anymore.